{
  "menu": {
    "exit": "끝내기",
    "help": "도움말을 확인",
    "completed": "완료",
    "language": "언어를 선택(CHOOSE LANGUAGE)",
    "cancel": "취소"
  },
  "language": {
    "_current": "지금 언어",
    "de": "독일어(Deutsch)",
    "en": "영어(English)",
    "fr": "프랑스어(français)",
    "ja": "일본어(日本語)",
    "zh-cn": "중국어(中文)",
    "zh-tw": "대만어(國語)",
    "es": "스페인어(Español)",
    "pt-br": "포르투갈어(Português)",
    "vi": "베트남어(Tiếng Việt)",
    "ru": "러시아어(Русский)",
    "ko": "한국어"
  },
  "error": {
    "exercise": {
      "none_active": "풀고 있는 연습문제가 없습니다. 메뉴에서 골라주세요.",
      "missing": "{{{name}}}(이)라는 연습문제는 없습니다.",
      "unexpected_error": "{{mode}}을(를) 할 수 없었습니다. {{{err}}}",
      "missing_file": "에러: {{{exerciseFile}}}은(는) 존재하지 않습니다!",
      "not_a_workshopper": "에러: {{{exerciseFile}}}은(는) 워크숍의 연습문제가 아닙니다.",
      "preparing": "연습문제의 준비에 실패했습니다. {{{err}}}",
      "loading": "연습문제의 택스트를 불러오는 중 에러가 발생했습니다. {{{err}}}"
    },
    "no_uncomplete_left": "지금의 연습문제를 마지막으로 더 이상 연습문제가 없습니다.",
    "cleanup": "청소 중 문제가 발생했습니다. {{{err}}}",
    "notty": "tty 호환 터미널에서 실행해주세요."
  },
  "solution": {
    "fail": {
      "title": "실패",
      "message": "{{{name}}}의 해결책은 통과하지 못했습니다. 다시 한번 해보세요!\n"
    },
    "pass": {
      "title": "통과",
      "message": "{{{name}}}의 해결책은 통과했습니다!"
    },
    "notes": {
      "compare": "비교해보고 싶다면 이 것이 공식 해답은 이렇습니다.\n",
      "load_error": "에러: 해답을 표시하는 도중 문제가 생겼습니다. {{{err}}}"
    }
  },
  "ui": {
    "return": "메뉴를 보려면 '{{appName}}'을(를) 입력하세요.\n",
    "usage": "사용법: {{appName}} {{mode}} mysubmission.js"
  },
  "progress": {
    "reset": "{{title}}의 진행 상황을 초기화했습니다.",
    "finished": "모든 연습문제를 끝냈습니다! 야호!\n",
    "state": "연습문제 {{amount}} 개 중 {{count}}개 째",
    "remaining": {
      "one": "한 개의 도전만 남아있습니다!",
      "other": "앞으로 {{count}} 개의 도전이 남아 있습니다."
    }
  }
}
